nlmod/mdims/resample.py

```diff
@@ -141,13 +137,13 @@ def resample_dataset_to_vertex_grid(ds_in, gridprops, method="nearest"):
 
     assert isinstance(ds_in, xr.core.dataset.Dataset)
 
-    xyi, _ = mgrid.get_xyi_icell2d(gridprops)
+    xyi, _ = get_xyi_icell2d(gridprops)
     x = xr.DataArray(xyi[:, 0], dims=("icell2d"))
```

Comment:
icell2d is not used, should it be?

Reply:
Yes, icell2d is not used, which is intended, we assume icell2d is always range(ncell2d), so we do not need to specify the coords of the xarray dataarray

Reply:
My bad, another old comment I guess, where icell2d was still assigned but not used.